From: Reinhard Tartler <siretart@tauware.de>
Date: Thu, 28 Aug 2025 16:02:19 -0400
Subject: Relax dependency on env_logger

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 1583bc3..913044c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,7 +35,7 @@ path = "src/test/main.rs"
 [dependencies]
 anyhow = "1.0.93"
 clap = { version = "~4.5.23", features = ["derive", "env"] }
-env_logger = "0.11.8"
+env_logger = "0.11.6"
 ipnet = { version = "2.11.0", features = ["serde"] }
 iptables = "0.5.2"
 libc = "0.2.157"
