From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:38:21 -0400
Subject: relax tonic dependency

Index: netavark/Cargo.toml
===================================================================
--- netavark.orig/Cargo.toml
+++ netavark/Cargo.toml
@@ -55,7 +55,7 @@ fs2 = "0.4.3"
 netlink-sys = "0.8.7"
 tokio = { version = "1.43.0", features = ["rt", "rt-multi-thread", "signal", "fs"] }
 tokio-stream = { version = ">= 0.1.16", features = ["net"] }
-tonic = "0.12.3"
+tonic = ">= 0.12"
 mozim = "0.2.5"
 prost = "0.12.6"
 futures-channel = "0.3.31"
@@ -67,7 +67,7 @@ hyper-util = "0.1.10"
 
 [build-dependencies]
 chrono = { version = "0.4.39", default-features = false, features = ["clock"] }
-tonic-build = "0.12.3"
+tonic-build = ">= 0.12"
 
 [dev-dependencies]
 once_cell = "1.20.2"
