From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:40:28 -0400
Subject: relax clap dependency

Index: netavark/Cargo.toml
===================================================================
--- netavark.orig/Cargo.toml
+++ netavark/Cargo.toml
@@ -32,7 +32,7 @@ deps-serde = ["chrono/serde", "url/serde
 
 [dependencies]
 anyhow = "1.0.93"
-clap = { version = "~4.5.23", features = ["derive", "env"] }
+clap = { version = ">= 4.4", features = ["derive", "env"] }
 env_logger = "0.11.6"
 ipnet = { version = "2.11.0", features = ["serde"] }
 iptables = "0.5.2"
