From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:40:49 -0400
Subject: relax env_logger dependency

Index: netavark/Cargo.toml
===================================================================
--- netavark.orig/Cargo.toml
+++ netavark/Cargo.toml
@@ -33,7 +33,7 @@ deps-serde = ["chrono/serde", "url/serde
 [dependencies]
 anyhow = "1.0.93"
 clap = { version = ">= 4.4", features = ["derive", "env"] }
-env_logger = "0.11.6"
+env_logger = { version = ">= 0.10" }
 ipnet = { version = "2.11.0", features = ["serde"] }
 iptables = "0.5.2"
 libc = "0.2.157"
